lib/modules/ftp/strong_password/files/ftp_sniffer.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
+
+import socket
+import _thread
+import threading
+from datetime import datetime
+import json
+
+users = {}
+HOST = ''
+PORT = 21
+LOGFILE = '/root/logs/ohp_ftp_strong_password_creds_logs.txt'
+LOGFILE_LOCK = threading.Lock()
+
+def init_user_conf():
+    f = open('/root/users.conf', 'r')
+    user_conf_lines = f.read().split('\n')
+    for user_conf_line in user_conf_lines:
+        split_line = user_conf_line.split(':')
+    if len(split_line) >= 2:
+        username = split_line[0]
+        password = split_line[1]
+        users[username] = password
+
+
+def init_server_conf():
+    f = open('/root/server.conf', 'r')
+    server_conf_lines = f.read().split('\n')
+    for server_conf_line in server_conf_lines:
+        split_line = server_conf_line.split('=')
+    if len(split_line) >= 2:
+        conf_variable = split_line[0]
+        conf_variable_value = split_line[1]
+        if conf_variable == 'PORT':
+            PORT = int(conf_variable_value)
+        elif conf_variable == 'HOST':
+            if conf_variable_value != 'DEFAULT':
+                HOST = str(conf_variable_value)
+
+
+def clientThread(conn, connip):
+    currentDir = "/"
```

Comment:
unused var?

Reply:
yup i removed the directory logic as we are not logging anything related to directory here.
forgot the remove the variable ,will update the script.